python/lsst/ts/cbp/csc.py

```diff
@@ -164,11 +171,13 @@ async def handle_summary_state(self):
             if self.component.parked:
                 await self.component.set_unpark()
         else:
+            self.telemetry_task.cancel()
+            await self.component.disconnect()
             if self.simulator is not None:
                 await self.simulator.close()
                 self.simulator = None
-            await self.component.disconnect()
-            self.telemetry_task.cancel()
+            # await self.component.disconnect()
```

Comment:
Is there a reason why these lines are commented out and why to keep them? If yes, please add a comment that justifies why they need to be kept and otherwise please remove them.